src/Uno.UI/Controls/Window.iOS.cs

```diff
@@ -340,6 +340,11 @@ public static void SetNeedsKeyboard(UIView view, bool needsKeyboard)
 
 		private static bool GetNeedsKeyboard(UIView view)
 		{
+			if (view == null)
+			{
+				return false;
+			}
+
 			var superViews = view.FindSuperviews().ToList();
 			superViews.Insert(0, view);
 
```

Comment:
I think the issue is likely to be on the line below, where `superViews` contains an array with an null entry, which then gets passed to `_attachedProperties.GetValue(..)` as the key. You may want to add this as well to remove nulls:

```suggestion
			var superViews = view.FindSuperviews().Trim().ToList();
			superViews.Insert(0, view);
```